foundry.toml

```diff
@@ -79,16 +79,6 @@
   out = "docs"
   repository = "https://github.com/sablier-labs/v2-core"
 
-[etherscan]
-  arbitrum = { key = "${API_KEY_ARBISCAN}" }
-  avalanche = { key = "${API_KEY_SNOWTRACE" }
-  bnb_smart_chain = { key = "${API_KEY_BSCSCAN}" }
-  gnosis_chain = { key = "${API_KEY_GNOSISSCAN}" }
-  mainnet = { key = "${API_KEY_ETHERSCAN}" }
-  optimism = { key = "${API_KEY_OPTIMISTIC_ETHERSCAN}" }
-  polygon = { key = "${API_KEY_POLYGONSCAN}" }
-  sepolia = { key = "${API_KEY_ETHERSCAN}" }
-
```

Comment:
@andreivladbrg why did you remove these? Was it because of the env var name mismatch? If yes, I fixed this with the separation between the base env and the deployment env.

Reply:
They are problematic and I always end up in passing `--etherscan-api-key` flag, thus I decided to remove them and I still consider to be the better choice

Reply:
I see. I agree that it would be helpful to remove the API keys from `foundry.toml`. They add maintenance cost.

> i find redundant and unnecessary to have 2 different .env files so i'd say to keep one

1. Not redundant because of security reasons; it's much much much safer not to load deployment keys all the time during testing and development
    - FYI: this separation between development and production env keys is super common in traditional software development
2. Reminder that necessary/ unnecessary is a bad criterion; helpful/ unhelpful is 1000x better

Reply:
> Reminder that necessary/ unnecessary is a bad criterion; helpful/ unhelpful is 1000x better

just wording, reformulating then:

i find redundant and unhelpful to have 2 different .env files so i'd say to keep one